### PR TITLE
selinux unit tests

### PR DESCRIPTION
**Pull request by radomil**
Tests related to #27

### CODE REVIEW

**Comment by radomil** [comment]
Tests fails due ImportError: No module named selinux

File: requirements.txt [hunk 1]
```diff
@@ -5,4 +5,5 @@ pytest==3.6.4
 pytest-ordering==0.5
 funcsigs==1.0.2
 mock==2.0.0
+selinux
```
Comment (vinzenz) [inline requirements.txt]: NO this is not ok, what you want is the Makefile with 'install-deps' target. Like this: https://github.com/oamg/leapp-repository/blob/master/repos/system_upgrade/el7toel8/actors/systemfacts/Makefile

File: repos/system_upgrade/el7toel8/actors/systemfacts/tests/test_systemfacts_selinux.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import selinux
+from leapp.libraries.actor.systemfacts import get_selinux_status
```
Comment (vinzenz) [inline repos/system_upgrade/el7toel8/actors/systemfacts/tests/test_systemfacts_selinux.py]: There should be an empty line between

File: requirements.txt (hunk 1 above)
Comment (radomil) [reply requirements.txt]: @vinzenz , I put libselinux-python to Makefile, still that hasn't help to the import.

**Comment by radomil** [comment]
@vinzenz , I've added the last test I wished to have in place. It's ready.

**Pull request merged**